src/stan/variational/advi.hpp

```diff
@@ -281,7 +438,7 @@ namespace stan {
               do_more_iterations = false;
             }
 
-            if (iter_counter > 2*eval_elbo_) {
+            if (iter_counter > 10*eval_elbo_) {
```

Comment:
I just looked at the cpplint python script. It looks like it's searching for an equal number of spaces around operators. I'm not sure how to configure it to enforce spaces.